jmeter/README.md

```diff
@@ -0,0 +1,48 @@
+# Agent Check: JMeter
+
+## Overview
+
+Datadog Backend Listener for Apache JMeter is a opensource JMeter plugin used to send test results to the Datadog platform. It provides real time reporting of test metrics like latency, the number of bytes sent and received and more. You can also send to Datadog complete test results as log entries.
+
+## Setup
+
+### Installation
+
+The Datadog Backend Listener plugin needs to be installed manually, you can find the latest release and more detailed installation instructions on [this page][1].
+
+### Configuration
+
+The plugin has the following configuration options:
+
+| Name       | Required | Default value | description|
+|------------|:--------:|---------------|------------|
+|apiKey | true | NA | Your Datadog API key.|
+|datadogUrl | false | https://api.datadoghq.com/api/ | You can configure a different endpoint, for instance https://api.datadoghq.eu/api/ if your datadog instance is in the EU|
```

Comment:
you can use a site selector for the docs site (find examples here https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1183023503/Regions). I am not sure how the declarative pipeline treats this though, for the app tiles

Reply:
Yeah it can't be used for tiles AFAIK